services/gcp/database/redis.yml

```diff
@@ -13,17 +13,9 @@ privileges:
     vulnerabilities: [destruction:data]
     notes: >-
       Deleting a Redis instance permanently removes all data stored within it.
-  get:
```

Comment:
I think we should keep the operations, just validate that the vulnerabilities are correct. Were you able to spin up a Redis instance and validate whether or not data read is possible using this API call / permission?

Reply:
I was able to verify the instance management and did not include any API-level operations. Hence remove them